NumLib/ODESolver/TimeDiscretization.h

```diff
@@ -142,6 +142,14 @@ class TimeDiscretization
     virtual void pushState(const double t, GlobalVector const& x,
                            InternalMatrixStorage const& strg) = 0;
 
+    /*! Indicate that the current timestep is rejected and that the computation
+     *  of the present time step will be restarted with a new time step size.
```

Comment:
Sounds a little misleading to me: The function does not indicate anything. Maybe:
> Restores previous time step result into the given vector.
> Used for repeating of the time step with new time step size when the current timestep is rejected.